R/deprecated.R

```diff
@@ -223,87 +76,18 @@ dist_spec <- function(distribution = c(
                         "lognormal", "normal", "gamma", "fixed", "empty"
                       ),
                       params_mean = numeric(0), params_sd = numeric(0),
-                      mean, sd = 0, mean_sd = 0, sd_sd = 0,
                       max = Inf, pmf = numeric(0), fixed = FALSE) {
 
-  lifecycle::deprecate_warn(
+  lifecycle::deprecate_stop(
```

Comment:
Won't this break the distribution interface since it's called internally? Seems like we want to use `deprecate_soft()` except it won't warn users. Seems convoluted to me.

Reply:
It shouldn't be called any more - `new_dist_spec()` is used internally to construct the objects.

Reply:
Got it!